agcaballero/portfolio/src/main/webapp/nav.html

```diff
@@ -0,0 +1,6 @@
+<ul id="nav-list" class="nav-ul">
+  <li class="nav-li"><a href="index.html">About</a></li>
+  <li class="nav-li"><a href="gallery.html">Gallery</a></li>
+  <li class="nav-li"><a href="blog.html">Blog</a></li>
+  <li class="nav-li" id="last-elem">AnneMarie Caballero</li>
+</ul>
```

Comment:
Adding a newline at the end avoids this warning.

Reply:
Done